### PR TITLE
The Nihilist

### PR DESCRIPTION
**Pull request by presheaf**
Straightforward implementation. Was unsure about how it should fit into phase 1.2, but I think there aren't that many cases where you care about the ordering anyway, so I elected to leave it for the big 1.2. cleanup. Let me know if you object.

### CODE REVIEW

File: src/clj/game/cards/resources.clj [hunk 1]
```diff
@@ -2027,6 +2027,35 @@
                                 :pre-breaker-strength {:req (req (= (:cid target)(:cid (:hai-target card))))
                                                        :effect (effect (breaker-strength-bonus 2))}}) card))}}
 
+   "The Nihilist"
+   (let [has-2-virus-tokens? (req (<= 2 (reduce + (map #(get-counters % :virus) (get-all-installed state)))))
```
Comment (NoahTheDuke) [inline src/clj/game/cards/resources.clj]: Not that it matters much, but is there any difference between `reduce +` and `+`?

Comment (presheaf) [reply src/clj/game/cards/resources.clj]: Oh, I forgot + can take multiple arguments. Then no!

Comment (presheaf) [reply src/clj/game/cards/resources.clj]: Oh, wait, I also forgot + doesn't take a list. Oops. So `reduce +` could become `apply +`.

Comment (Saintis) [reply src/clj/game/cards/resources.clj]: Well you already have a map, so maybe you want something like `(reduce %(+ %1 (get-counters %2 :virus)) 0 (get-all-installed state))`.

Comment (Saintis) [reply src/clj/game/cards/resources.clj]: Ah, yes, using an existing function seems much better :)

**Pull request merged**